src/plugins/intel_gpu/src/graph/impls/ocl/roi_align.cpp

```diff
@@ -1,25 +1,43 @@
 // Copyright (C) 2018-2022 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
-#include "roi_align_inst.h"
+
+// We have problem with includes when ENABLE_ONEDNN_FOR_GPU is OFF,
+// "impl_types" enum is not accessible if "implementation_map.hpp" is included first
+// so, a "fix" for now is to turn off clang-format for these include
+// clang-format off
 #include "primitive_base.hpp"
 #include "impls/implementation_map.hpp"
+// clang-format on
 #include "intel_gpu/runtime/error_handler.hpp"
 #include "kernel_selector_helper.h"
-#include "roi_align/roi_align_kernel_selector.h"
 #include "roi_align/roi_align_kernel_ref.h"
+#include "roi_align/roi_align_kernel_selector.h"
+#include "roi_align_inst.h"
 
 namespace cldnn {
 namespace ocl {
 
 namespace {
 kernel_selector::pool_type from(roi_align::PoolingMode mode) {
     switch (mode) {
-        case roi_align::PoolingMode::Max:
-            return kernel_selector::pool_type::MAX;
-        default:
-        case roi_align::PoolingMode::Avg:
-            return kernel_selector::pool_type::AVG;
+    case roi_align::PoolingMode::Max:
+        return kernel_selector::pool_type::MAX;
+    default:
+    case roi_align::PoolingMode::Avg:
+        return kernel_selector::pool_type::AVG;
+    }
+}
+
+kernel_selector::roi_aligned_mode from(roi_align::AlignedMode mode) {
+    switch (mode) {
+    case roi_align::AlignedMode::Half_pixel_for_nn:
+        return kernel_selector::roi_aligned_mode::Half_pixel_for_nn;
+    case roi_align::AlignedMode::Half_pixel:
+        return kernel_selector::roi_aligned_mode::Half_pixel;
```

Comment:
to be consistent with other enums in kernel_selecter better to have naming like HALF_PIXEL (as MAX and AVG)

Reply:
Done